readme.md

```diff
@@ -71,6 +71,7 @@ Click the type names for complete docs.
 - [`RequireAtLeastOne`](source/require-at-least-one.d.ts) - Create a type that requires at least one of the given properties.
 - [`ReadonlyDeep`](source/readonly-deep.d.ts) - Create a deeply immutable version of a `object`/`Map`/`Set`/`Array` type.
 - [`LiteralUnion`](source/literal-union.d.ts) - Create a union type by combining primitive types and literal types without sacrificing auto-completion in IDEs for the literal type part of the union. Workaround for [Microsoft/TypeScript#29729](https://github.com/Microsoft/TypeScript/issues/29729).
+- [`Promisable`](source/promisable.d.ts) - Create a type for values that can either be a promise or it's return value, and as such both accepts synchronous values and promise-likes of those values.
```

Comment:
I still don't think this is clear enough. Couldn't it simply be:

> Create a type that represents either the value or the value wrapped in `Promise`.

Reply:
I think that would indeed be more readable.

Reply:
👍